spec/requests/network_routers_spec.rb

```diff
@@ -204,4 +204,36 @@
       expect(response.parsed_body).to include(expected)
     end
   end
+
+  describe 'OPTIONS /api/network_routers' do
+    it 'with ems_id="..." returns a DDF schema for add when available via OPTIONS' do
+      provider = FactoryBot.create(:ems_network)
+
+      allow(provider.class::NetworkRouter).to receive(:params_for_create).and_return('foo')
+
+      options("#{api_network_routers_url}?ems_id=#{provider.id}")
+
+      expect(response.parsed_body['data']['form_schema']).to eq('foo')
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'with no ems_id returns no data' do
+      options(api_network_routers_url.to_s)
+
+      expect(response.parsed_body['data']).to eq({})
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe 'OPTIONS /api/network_routers/:id' do
+    it 'returns a DDF schema for edit when available via OPTIONS' do
+      network_router = FactoryBot.create(:network_routers)
+      allow(NetworkRouter).to receive(:find).with(network_router.id.to_s).and_return(network_router)
+      allow(Rbac).to receive(:filtered_object).and_return(network_router)
+      expect(network_router).to receive(:params_for_edit).and_return('foo')
```

Comment:
elsewhere I believe we called this params_for_update.  @agrare Can you confirm?

Reply:
That's correct CloudVolumes calls this `params_for_update`

Reply:
I can change it to params_for_update